ACPs/30-avalanche-warp-x-evm.md

```diff
@@ -140,19 +140,30 @@ The execution gas costs were determined by summing the cost of the EVM operation
 
 ##### sendWarpMessage
 
-`sendWarpMessage` charges a base cost of 41,875 gas
+`sendWarpMessage` charges a base cost of 41,500 gas + 8 gas / payload byte
 
 This is comprised of charging for the following components:
 
 - 375 gas / log operation
-- 4 topics * 375 gas / topic
+- 3 topics * 375 gas / topic
 - 20k gas to produce and serve a BLS Signature
 - 20k gas to store the Unsigned Warp Message
+- 8 gas / payload byte
 
 This charges 20k gas for storing an Unsigned Warp Message although the message is stored in an independent key-value database instead of the active state. This makes it less expensive to store, so 20k gas is a conservative estimate.
 
 Additionally, the cost of serving valid signatures is significantly cheaper than serving state sync and bootstrapping requests, so the cost to validators of serving signatures over time is not considered a significant concern.
 
+`sendWarpMessage` also charges for the log operation it includes commensurate with the gas cost of a standard log operation in the EVM.
+
+A single `SendWarpMessage` log is charged:
+
+- 375 gas base cost
+- 375 gas per topic (`EventID`, `sender`, `messageID`)
```

Comment:
nit: why is `EventID` capitalized but the others aren't?

Reply:
The real reason is `EventID` is how it's capitalized in the code whereas `sender` / `messageID` are in the solidity interface where it starts with lowercase and my brain just copy pasted that context into this doc.

Updating so that it's uniform capitalization on this line.